.template/addons/svgeez/.github/wiki/template.rb

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
This extra line break should be removed

Reply:
Thanks. Fixed in 5685935937d6e52e31976dd93d9506de59105ca1